.gitignore

```diff
@@ -1,3 +1,4 @@
 /bower_components/
 /node_modules/
 /tmp/
+/tests/
```

Comment:
I don't think this is correct.  I definitely do not want to ignore `tests` in git.  Maybe you meant `.npmignore`?